spec/person_spec.rb

```diff
@@ -0,0 +1,23 @@
+require_relative '../Person/nameable'
+require_relative '../Person/rental'
+require_relative '../Person/person'
+
+RSpec.describe Person do
+  let(:age) { 20 }
+  let(:name) { 'Jane Doe' }
+  let(:person) { Person.new(age, name) }
+
+  describe 'initailize' do
+    it 'creates a new person object' do
+      expect(person.age).to eq(age)
+      expect(person.name).to eq(name)
+      expect(person.rentals).to be_empty
+    end
+  end
+
+  describe 'correct name' do
+    it 'returns persons name' do
+      expect(person.correct_name).to eq(name)
+    end
+  end
```

Comment:
- [ ] Great job so far 👏🏻 👏🏻 However, according to the project requirements, there should be unit tests for all the methods in all the classes, Kindly make sure the `can_use_services?`, `of_age?`, and  `add_rental` class person method has (at least) an 'it' block 👇🏻 check your person.rb file

```ruby 
  def can_use_services?
    of_age? || @parent_permission
  end

  private

  def of_age
    @age.to_i >= 18
  end

  def add_rental(rental)
    @rentals.push(rental)
    rental.person = self
  end
```